.github/workflows/release.yml

```diff
@@ -51,8 +53,6 @@ jobs:
 
       - name: Bump version
         run: yarn version --${{ github.event.inputs.version_bump }}
```

Comment:
`yarn version` should run before `yarn build` because the esm.js file has a comment on top stating the version of the script. @Orkuncakilkaya